packages/eslint-config/shared.js

```diff
@@ -143,6 +143,7 @@ module.exports = {
         ],
       },
     ],
+    '@typescript-eslint/consistent-type-imports': 'error',
```

Comment:
@virtuoushub should this rule go in shared? That means that it'll apply to Redwood projects and since it's set to err, that feels breaking

Reply:
good catch! nope. just fixed this.